cfn-resources/cfn-publishing-automation/cfn-3P-provider-register.yaml

```diff
@@ -242,7 +235,7 @@ mainSteps:
         - FAULT
         - STOPPED
         - TIMED_OUT
-      timeoutSeconds: 7200
+    timeoutSeconds: 10800
```

Comment:
I don't know, I'm just copying the last version in the AWS account to have it under version control. I guess it'll be because tests that take time to create clusters and then snapshots or long operations